templates/emails/setup.txt

```diff
@@ -21,10 +21,8 @@ Or you can forward incoming mail to your private email account by following thes
 
 Your website address is http://djangogirls.org/{{ event.eventpage.url }}
 
-Login here:
-http://djangogirls.org/admin/
-{% for member in members %}{{ member.first_name }} - email: {{ member.email }} password: {{ member.password }}
-{% endfor %}
+You should just receive a separate email with auto generated password and instructions to login. 
```

Comment:
Should be: "You should receive (...) in a moment" or "We have just sent you an e-mail" or "You should have just received" :)